### ISSUE TITLE
Sample questions - Answers - Answer key: Line number overlaps text 

### ISSUE THREAD

**Issue by mhamburg** [istqborg/istqb_product_base#159]
The line number '111' of the second column overlaps the text of the first column.

<img width="438" alt="Screenshot 2024-12-30 110935" src="https://github.com/user-attachments/assets/74093e52-66a2-4eee-a7b7-b7b72f676010" />

**Comment by danopolan** [comment]
I wasn't able to reproduce this issue in latest build https://github.com/istqborg/istqb-ctal-ta/actions/runs/12525438677
Closing.

**Comment by mhamburg** [comment]
Here it comes again: 
<img width="393" alt="Screenshot 2025-01-12 162144" src="https://github.com/user-attachments/assets/0c7121a5-9c86-46cf-b4ad-d00b2d16f8d0" />

**Comment by danopolan** [comment]
Matthias, this is about line numbering being turned on and we cannot do much about it, since table on the right is also numbered as a line. But I assume this does not have an effect on the release.

**Comment by mhamburg** [comment]
I agree that this will not affect the release. And I agree that it is a cosmetic defect.
That said, I am sure you can do something about it. It is just a matter of the cost/benefit relation.
